platforms/hono.ts

```diff
@@ -21,13 +21,13 @@ export type WorkflowBindings = {
  */
 export const serve = <
   TInitialPayload = unknown,
-  // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-parameters
   TBindings extends WorkflowBindings = WorkflowBindings,
+  TVariables extends object = object,
```

Comment:
on a type-level the exact same, but I suggest using the `Variables` type provided directly by Hono:

`import { Variables } from "hono/types"`